src/components/CheckList/CheckListItemPanel.tsx

```diff
@@ -4,14 +4,14 @@ import { Label, Text, makeStyles } from "@fluentui/react-components";
 import { FunctionComponent } from "react";
 import { FluentProvider, webLightTheme } from "@fluentui/react-components";
 import { InfoIcon, TextFieldIcon } from "@fluentui/react-icons-mdl2";
-import { addHook, sanitize } from "dompurify";
+import DOMPurify from "dompurify";
```

Comment:
I was wondering if this was going to cause issues, as I am stepping through your commits, looks like it did.